libs/blocks/locui-create/input-actions/modal.js

```diff
@@ -9,7 +9,7 @@ function Modal() {
       <strong>Project "${project.value.name || 'n/a'}" Successfully Created</strong>
     </div>
     <div class="create-project-view">
-     <button class="s2-btn accent">View</button>
+    ${projectInfo.value.projectLink && html`<a class="s2-btn accent" href="${projectInfo.value.projectLink}" target="_blank" rel="noreferrer noopener">View</a>`}
```

Comment:
You may want to use `projectInfo?.value?.projectLink`

Reply:
Hi @nkthakur48 , 
I checked with @raga-adbe-gh on this, he said that we need to use project link to go to the project details screen, as the preview url is not available at that time.